svf/include/SVFIR/SVFValue.h

```diff
@@ -284,6 +285,7 @@ class SVFFunction : public SVFValue
     SVFLoopAndDomInfo* loopAndDom;  /// the loop and dominate information
     const SVFFunction* realDefFun;  /// the definition of a function across multiple modules
     std::vector<const SVFBasicBlock*> allBBs;   /// all BasicBlocks of this function
+    std::vector<SVFBasicBlock*> allExtBBs;   /// all BasicBlocks of this external function
```

Comment:
Do we need this data structure? If so, we will need to change the SVFIR json as well @xudon9

Reply:
Because external functions only have a declaration but no definition, if we want to add a CallStmt, we need to create a BasicBlock within the external function and then add the CallStmt to the BasicBlock. Furthermore, the BasicBlock cannot be 'const'.

Reply:
Why do we want exitBBs?

Reply:
can we use allBBs or not?

Reply:
The "ext" here means External I think.

Reply:
Since an external function does not have a body. Could we use allBBs to include the created BB?

Reply:
I try to see if it is possible to only use allBBs.